signer/tests/integration/postgres.rs

```diff
@@ -1265,3 +1265,56 @@ async fn we_can_fetch_bitcoin_txs_from_db() {
 
     signer::testing::storage::drop_db(pg_store).await;
 }
+
+async fn transaction_coordinator_test_environment(
+) -> testing::transaction_coordinator::TestEnvironment<impl FnMut() -> storage::postgres::PgStore> {
+    use std::sync::atomic::Ordering;
+
+    let test_model_parameters = testing::storage::model::Params {
+        num_bitcoin_blocks: 20,
+        num_stacks_blocks_per_bitcoin_block: 3,
+        num_deposit_requests_per_block: 5,
+        num_withdraw_requests_per_block: 5,
+        num_signers_per_request: 7,
+    };
+
+    let db_num = testing::storage::DATABASE_NUM.fetch_add(1, Ordering::SeqCst);
+    let store = testing::storage::new_test_database(db_num, true).await;
+    // TODO: something better than this hacky thing?
+    let mut store_vec = vec![store];
+
+    testing::transaction_coordinator::TestEnvironment {
+        storage_constructor: move || store_vec.pop().unwrap(),
```

Comment:
Not proud of this, do you have anything smarter to apply here?

Reply:
Not that I'm aware. I tried and couldn't come up with anything.

Reply:
This goes away in #591 so I wouldn't worry about it